vmdb/app/models/ems_microsoft.rb

```diff
@@ -43,6 +43,9 @@ def connect(options = {})
   end
 
   def verify_credentials(_auth_type = nil, options = {})
+    silence_warnings { require 'winrm' }
+    silence_warnings { require 'gssapi' } # Version 1.0.0 of the gssapi gem emits warnings
```

Comment:
Note, we definitely need this because require 'winrm' doesn't require gssapi, see [here](https://github.com/ManageIQ/manageiq/blob/899734a8a8df29dd1edbf33e75c02ba8bbd6c97b/vmdb/app/models/ems_microsoft.rb#L24)